packages/frontend-2/components/workspace/invite/DiscoverableWorkspaceBanner.vue

```diff
@@ -60,13 +65,31 @@ const processJoin = async (accept: boolean) => {
     return
   }
 
+  const userId = activeUser.value?.id
+  if (!userId) return
+
   const result = await apollo
     .mutate({
       mutation: DashboardJoinWorkspaceDocument,
       variables: {
         input: {
           workspaceId: props.workspace.id
         }
+      },
+      update(cache) {
+        modifyObjectField(
```

Comment:
You should always use the 2nd param of `update` - `data` - to get the actual data from the response and validate that the mutation did in fact go through successfully before you make any cache updates